packages/components/src/components/Table/Td.tsx

```diff
@@ -15,7 +15,19 @@ export interface TdProps
 const Td = React.forwardRef<HTMLTableCellElement, TdProps>(
   ({ children, colSpan, rowSpan, ...props }, ref) => {
     return (
-      <Box.td colSpan={colSpan} ref={ref} rowSpan={rowSpan} {...props}>
+      <Box.td
+        colSpan={colSpan}
+        fontSize="fontSize20"
+        fontWeight="fontWeightRegular"
```

Comment:
Let's not set font size here. That can have an affect on child elements. The font size can be set on those child elements.